examples/demo/data/numeric/signalProfiles.py

```diff
@@ -0,0 +1,107 @@
+import numpy as np
+from scipy import signal as sig
+
+def binaryProfile(freqFact=None,
```

Comment:
please try to add some docstrings for all functions that you defined in this document

Reply:
added